components/careers/HeroSection.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <section>
+    <v-container>
+      <v-stack
+        direction="column"
+        md-direction="column"
+        sm-direction="column"
```

Comment:
I guess we can use just `direction` prop if the value is not changing in any resolution. Can you try it and check if it works? If it works, we won't require `md-direction` and `sm-direction`. Not sure if it's required for `md-gap` and `sm-gap`